WPFHospitalEditor/Controller/MapObjectController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using WPFHospitalEditor.MapObjectModel;
+using WPFHospitalEditor.Service;
+
+namespace WPFHospitalEditor.Controller
+{
+    class MapObjectController : IMapObjectController
+    {
+        private MapObjectService MapObjectService { get; set; }
+
+        public MapObjectController(MapObjectService mapObjectService)
```

Comment:
Change parameter to be instance of interface. Check Dependency Injection by constructor.

Reply:
Controller, Service and Repository classes are changed and this is fixed.